### ISSUE TITLE
Vpinmame 64 bit Baby Pacman Roms do not run correctly, 32 bit they do but not 64 bit.

### ISSUE THREAD

**Issue by psiomicron** [vpinball/pinmame#177]
Vpinmame 64 bit Baby Pacman Roms do not run correctly, get garbled graphics.

32 bit they do but not 64 bit. 

Running latest Vpinmam64 as of 10/22/23

**Comment by toxieainc** [comment]
Duplicate of #80

**Comment by bismark-0610** [comment]
Seems like the latest 32 bit version has issues. The drop targets are acting funny. When you hit the drop target to allow you to hit the upper ball, it keeps resetting automatically.

**Comment by sosim5** [comment]
Yeah, I mentioned the drop target problem in another thread. The drop target in front of the captive ball is up where it should be down. I'm just not sure if its a VPX or VPinMame problem. Last time it worked on VPX was 10.8.0-925

**Comment by bismark-0610** [comment]
Looks like it might actually be a VPX issue? I went back to 10.7.4 and the issue is not there while still running with the latest 32bit pinmame.

**Comment by JON99999** [comment]
I can confirm this bug goes back to 2.8, (I think, if I did the backwards installs properly).  I too am able to make it run properly on 32bit 10.7.4.  My setup is a standard 1st position display for playfield at 1920x1080 (rotated by vpinball) and 2nd position 4:3 monitor in standard horizontal output.  (1024x768).  (I didn't document, but curiously, it seemed like different versions show different corruptions sometimes.  like, the pixels are slightly different positions or colors?  There's a good change though this was just where the actual image was when I happened to look.)

**Comment by JON99999** [comment]
Also, if you read the whole thread of 32Assassin's original table post, the issues with the drop targets are discussed heavily and some of the folks made a different version to fix it but didn't release publicly.  https://www.vpforums.org/index.php?showtopic=38512
